doc/dune-files.rst

```diff
@@ -1497,6 +1497,29 @@ run this toplevel with:
 
    $ dune exec ./tt.exe
 
+chdir
+-----
+
+The ``chdir`` stanza can be used to evaluate stanzas in sub directories. This is
+useful for generated files or to override stanzas in vendored direcotries
+without editing vendored dune files.
```

Comment:
Is the doc up-to-date? I thought we settled on no shadowing.

Reply:
Doc removes the example of shadowing. Note that it wasn't really an example of shadowing as discussed earlier. It was just using the `data_only_dirs` stanza to ignore a dune file, so it was completely explicit.